docs/docs/component-css.md

```diff
@@ -2,7 +2,192 @@
 title: Component CSS
 ---
 
-This is a stub. Help our community expand it.
+### Component CSS
 
-Please use the [Gatsby Style Guide](/docs/gatsby-style-guide/) to ensure your
-pull request gets accepted.
+Gatsby has a wealth of options available for styling components. This guide will explore one very popular method: CSS Modules. If you would like to see other CSS options, check out the bottom of the page: [Other CSS Options](#other-css-options).
+
+### CSS Modules
```

Comment:
Hm, I know that this issue specifically instructed you to copy and paste from the tutorial. Now that I see it, I'm realizing that linking to the tutorial will be the easiest to maintain down the road. Apologies for the change! Will you replace this copied/pasted content with a link to the tutorial and then a very basic code example? It will make the guide fairly short, which is fine!

Reply:
Will do